plugins/language_lilypond.lua

```diff
@@ -0,0 +1,21 @@
+-- mod-version:3
+local syntax = require "core.syntax"
+
+syntax.add {
+  name = "LilyPond",
+  files = { "%.i?ly$" },
+  comment = "%%",
```

Comment:
```suggestion
  comment = "%%",
  block_comment = { "%%{", "%%}" },
```
This enables the command to add block comments.